evm/vm/forks/frontier/state.py

```diff
@@ -0,0 +1,82 @@
+from evm import constants
+from evm.exceptions import (
+    OutOfGas,
+    InsufficientFunds,
+    StackDepthLimit,
+)
+from evm.state import (
+    BaseState
+)
+from evm.utils.hexadecimal import (
+    encode_hex,
+)
+from evm.utils.keccak import (
+    keccak,
+)
+
+
+class FrontierState(BaseState):
+    def apply_message(self, message):
```

Comment:
oooh, I think I like that we can now declare all of these methods on an actual object!